src/pipeline.h

```diff
@@ -182,6 +184,8 @@ struct PipelineBaton {
     tiffCompression(VIPS_FOREIGN_TIFF_COMPRESSION_JPEG),
     tiffPredictor(VIPS_FOREIGN_TIFF_PREDICTOR_NONE),
     tiffSquash(false),
+    tiffXres(10),
```

Comment:
It looks like libvips uses a default of `1.0` - see https://github.com/jcupitt/libvips/blob/d1ca808769339a8856be69b281b72d674a9c0f75/libvips/foreign/